qemu/tests/block_copy.py

```diff
@@ -136,6 +136,40 @@ def is_cancelled():
         if self.vm.monitor.protocol == "qmp":
             self.vm.monitor.clear_event("BLOCK_JOB_CANCELLED")
 
+    def is_paused(self):
+        """
+        Return block job paused status.
+        """
+        paused = self.get_status().get("paused")
+        offset_p = self.get_status().get("offset")
+        time.sleep(random.uniform(1, 3))
+        offset_l = self.get_status().get("offset")
+        paused &= offset_p == offset_l
```

Comment:
This is not work with rhel6 host.  As in rhel6 host, "query-block-jobs" has no option "paused", so in the code,
the variable "paused" is NoneType,  and  "paused &= offset_p == offset_1" will failed as "TypeError: unsupported operand type(s) for &=: 'NoneType' and 'bool'" 

rhel 6.8:
{"execute": "query-block-jobs", "id": "vix3JVGK"}
{"return": [{"device": "drive_image1", "len": 21474836480, "offset": 14697, "speed": 0, "type": "mirror"}], "id": "vix3JVGK"}

rhel 7.3:
{"execute": "query-block-jobs", "id": "vmaMoZC0"}
{"return": [{"io-status": "ok", "device": "drive_image1", "busy": true, "len": 3719692288, "offset": 52494336, "paused": false, "speed": 0, "ready": false, "type": "mirror"}], "id": "vmaMoZC0"}